diff --git a/src/azure-keyvault-slot.cpp b/src/azure-keyvault-slot.cpp
--- a/src/azure-keyvault-slot.cpp
+++ b/src/azure-keyvault-slot.cpp
@@ -10,7 +10,6 @@
 #include <azure/identity/chained_token_credential.hpp>
 #include <azure/identity/managed_identity_credential.hpp>
 #include <azure/identity/client_secret_credential.hpp>
-#include <azure/identity/azure_cli_credential.hpp>
 #include <azure/identity/workload_identity_credential.hpp>
 
 #include <openssl/x509.h>
@@ -93,9 +92,7 @@ std::shared_ptr<Azure::Core::Credentials::TokenCredential> get_credential()
     Azure::Identity::ChainedTokenCredential::Sources{
         std::make_shared<Azure::Identity::EnvironmentCredential>(),
         std::make_shared<Azure::Identity::WorkloadIdentityCredential>(),
-        getClientSecretCredential(),
-        std::make_shared<Azure::Identity::AzureCliCredential>(),
-        std::make_shared<Azure::Identity::ManagedIdentityCredential>()});
+        getClientSecretCredential()});
     return chainedTokenCredential;
 }
 
